client/src/views/page-components/ChatDetailPage/ChatDetaiPageHeader/ChatDetailPageHeader.js

```diff
@@ -0,0 +1,31 @@
+import { LEFT_ICON, LOGOUT_ICON } from '../../../../constants/imagePath.js';
+import { createDOMwithSelector } from '../../../../utils/createDOMwithSelector.js';
+
+export default function ChatDetailPageHeader({ $target, onClickOutHandler }) {
+  this.$header = createDOMwithSelector('div', '.chatDetailPageHeader');
+  this.onClickOutHandler = onClickOutHandler;
+  this.$header.addEventListener('click', (e) => {
+    const link = e.target.closest('button')?.dataset.link;
+    if (!link) return;
+    if (link === 'back') {
+      history.back();
+    } else if (link === 'out') {
+      this.onClickOutHandler();
+    }
+  });
+  $target.appendChild(this.$header);
+
+  this.render = () => {
+    this.$header.innerHTML = `
+      <button class="chatDetailPageHeader__back__container" data-link="back">
+        <img class="chatDetailPageHeader__icon" src="${LEFT_ICON}"/>
+      </button>
+      <span class="chatDetailPageHeader__title">${'유저이름'}</span>
```

Comment:
추후에 변수로 받기 위해 ${}를 사용하신 걸까요 :)??

```suggestion
      const USER_NAME = '유저이름'
      <span class="chatDetailPageHeader__title">${USER_NAME}</span>
```

Reply:
앗 맞습니다! ㅎㅎ
말씀대로 변수로 표현해서 변경해놓는게 좋겠네요!